lib/awful/hotkeys_popup/keys/tmux.lua

```diff
@@ -0,0 +1,73 @@
+---------------------------------------------------------------------------
+--- tmux hotkeys for awful.hotkeys_widget
+--
+-- @autor nahsi nashi@airmail.cc
+-- @copyright 2017 nahsi
+-- @module awful.hotkeys_popup.keys.tmux
+---------------------------------------------------------------------------
+
+local hotkeys_popup = require("awful.hotkeys_popup.widget")
+
+local tmux_rule = { name = "tmux" }
```

Comment:
but it's never like that even with default tmux config

as a solution to this a have in mind only to turn lines 11-19 into a function which will have rule as an argument, so any user can call it with any arguments to match the terminal

for example, in my case it would be like that:

```
rc.lua
<...>
local tmux_keys = require("awful.hotkeys_popup.keys.tmux")
tmux_keys.add_terminal_rule({class="st-256color"})
<...>
```